AutoZI_tutorial.ipynb

```diff
@@ -1,757 +1,764 @@
 {
```

Comment:
<p>let's remove this final cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>